db/version_set.cc

```diff
@@ -5186,6 +5188,8 @@ Status VersionSet::ProcessManifestWrites(
   autovector<Version*> versions;
   autovector<const MutableCFOptions*> mutable_cf_options_ptrs;
   std::vector<std::unique_ptr<BaseReferencedVersionBuilder>> builder_guards;
+  std::vector<const std::vector<uint64_t>*> files_to_quarantine_if_commit_fail;
+  std::vector<uint64_t> limbo_descriptor_log_file_number;
```

Comment:
Does `autovector` work for these types?

Reply:
`autovector` works well. Thank you for the suggestion!